grafana-tikv-dashboard.md

```diff
@@ -42,6 +42,7 @@ This document provides a detailed description of these key metrics on the **TiKV
 - gRPC message error: The number of gRPC message errors per type on each TiKV instance
 - Leader drop: The count of dropped leaders per TiKV instance
 - Leader missing: The count of missing leaders per TiKV instance
+- Log Replication Reject: The number of logappend messages rejected due to insufficient memory on each TiKV instance (update the picture later --to-be-deleted)
```

Comment:
```suggestion
- Log Replication Reject: The number of log append messages rejected due to insufficient memory on each TiKV instance
```

Reply:
'logappend' here means a message type in raft, 'logappend' may be better than 'log append'.